src/scheduler/workers/processListsBeforeAndDuringStart/main.ts

```diff
@@ -145,8 +136,8 @@ export async function processList(list: List, listItemsForList: ListItemWithHist
     return;
   }
   // get the most recent audit record to determine if the email has already been sent for the respective milestones
-  const audit = await getLatestReminderAuditEvent(annualReviewRef, "list");
-  const listItemAudit = await getLatestReminderAuditEvent(annualReviewRef, "listItem");
+  const audit = await getLatestReminderEvent(async () => await findReminderAuditEvents(annualReviewRef));
+  const listItemAudit = await getLatestReminderEvent(async () => await findAllReminderEvents(annualReviewRef));
```

Comment:
I think this may be more trouble than it's worth.. maybe we just call the find functions outside the `getLatestReminderEvent` scope, then pass in the result to `getLatestReminderEvent`?

```
const { result: allAuditEvents } = findReminderAuditEvents(..)
const { result: listItemEvents } = findAllReminderEvents(..)
const latestAudit = getLatestReminderEvent(allAuditEvents)
const latestEvent ...

```